content/tutorials/get-started/set-up-secure-signing.md

```diff
@@ -0,0 +1,134 @@
+
+3. Maintain the server to keep it running, updated, and in sync with the network while you're using it.
+
+    **Note:** You _can_ turn off your `rippled` server when you're not sending transactions, but it can take up to 15 minutes to sync with the network when you start it up again.
+
+
+## Run rippled on the same LAN
+
+[![Diagram of using a rippled server over LAN for signing](img/secure-signing-lan-rippled.png)](img/secure-signing-lan-rippled.png)
+
+In this configuration, you run a `rippled` server on a dedicated machine in the same private local area network (LAN) as the machine that generates the transactions to be signed. This configuration lets you assemble transaction instructions on one or more machines with very modest system specs, while using a single dedicated machine for running `rippled`. This may appeal to you if you run your own datacenter or server room.
+
+As always, follow industry-standard practices for securing your machines, such as using a firewall, anti-virus, appropriate user permissions, and so on.
+
+**Warning:** This configuration comes with the additional downside that anyone on the LAN can sniff traffic between your machines, potentially gaining access to your secret keys. Do not use this configuration on a network that may have strangers on it. For example, on the LAN at a colocation facility or cloud hosting, other customers may be able to get access to the traffic between your machines. If you employ several developers sending test transactions, you could run one `rippled` machine for your whole office, while the developers use cheaper hardware, but any user on your office network could potentially use a packet sniffer to get access to developers' secret keys.
+
+To use this configuration:
+
+1. [Install `rippled`](install-rippled.html) on the chosen machine.
+
+    Be sure that this machine meets the minimum [system requirements for `rippled`](system-requirements.html).
+
+2. Configure the `rippled` machine to have a static IP address in the local private LAN, or use a private DNS server to give the server a consistent name.
+
+    With IPv4, private LAN addresses commonly start with `10.`, `192.168.`, or `172.(16 to 31).`. With IPv6, private LAN addresses typically start with `fc` or `fd`.
+
+3. Configure the `rippled` machine to accept connections from your local private network.
+
+    In the `rippled`'s config file, modify the `[port_rpc_admin_local]` and `[port_ws_admin_local]` stanzas to use your server's private-LAN address:
+
+        [port_rpc_admin_local]
+      	port = 5005
+      	ip = 10.1.0.0 # Changed from 127.0.0.1
+      	admin = 10.1.1.2 # Only the client with this IP is treated as admin
+      	protocol = http
```

Comment:
shouldn't we still recommend `https/wss` for this LAN config?

Reply:
Good question. I copied the stanza from the example config and wrote text with the assumption that the communications would be unencrypted, but if https/wss is an option, that would be an improvement.

The question, of course, is the certificates. I don't think having `rippled` use self-signed certs here is a good idea. But setting up rippled to use a proper certificate (assuming that's even possible) from either a well-known CA or an internal CA is probably complicated enough to warrant a separate tutorial.

Reply:
We should avoid describing this setup _without_ referencing WSS/HTTPS. We should recommend that users install real SSL certificates, issued by a CA and link to Let's Encrypt as one such authority.

Reply:
ssl certs on a local network seem meh...I mean they are fine, but sure is a lot of work for not much gain. I see value in enabling ssl, just not convinced that certs on the 10. network buy you all that much unless you think something on your LAN can hijack the 10. . That said, I'm not an expert and will defer to Nik et. al.

Reply:
I'm kicking this can down the road to a future task. I simplified the instructions for now, and left a draft page to be expanded later.